package.json

```diff
@@ -1,56 +1,56 @@
 {
```

Comment:
it is better to keep the original format

Reply:
fixed